src/components/SearchBar/style.ts

```diff
@@ -11,7 +11,8 @@ export const SearchInput = styled.input`
   width: 100%;
   padding: 1rem;
   border: none;
-  font-size: 1rem;
+  /* absolute px - for Remove iOS zoom in */
+  font-size: 16px;
```

Comment:
```suggestion
font-size: 1rem;

@media screen and (max-width: 767px) {
    font-size: 16px;
}
```

이런식으로 모바일 화면에서만 px 사이즈로 변경되게 하는건 어떨까요?

Reply:
`src/styles/GlobalStyle.tsx` 를 확인해보시면 px이 변하여, 
rem에 영향을 주는 width는 아래와 같습니다.

```ts
html {
    @media (max-width: 599px) {
      font-size: 15px;
    }
    @media (max-width: 520px) {
      font-size: 14px;
    }
    @media (max-width: 470px) {
      font-size: 13px;
    }
    @media (max-width: 430px) {
      font-size: 12px;
    }
    @media (max-width: 400px) {
      font-size: 11px;
    }
    @media (max-width: 370px) {
      font-size: 10px;
    }
    @media (max-width: 330px) {
      font-size: 9px;
    }
    @media (max-width: 300px) {
      font-size: 8px;
    }
  }
```

해당 의견을 반영하여 아래 코드로 변경하겠습니다

```ts
@media (max-width: 600px) {
    font-size: 16px;
  }
```